buildroot-external/board/raspberrypi/rpi5-64/config.txt

```diff
@@ -43,6 +43,12 @@ arm_boost=1
 os_prefix=slot-A/
 cmdline=/cmdline.txt
 
+# Uncomment this to enable GPIO support for RPI-RF-MOD/HM-MOD-RPI-PCB
+#enable_uart=1
+#dtparam=i2c_arm=on
+#dtoverlay=miniuart-bt
```

Comment:
@jens-maus Is this still needed on RPi5? AFAIK it has a dedicated UART port now - but is it still multiplexing it with BT pins?

Reply:
@sairon I just tested if `dtoverlay=miniuart-bt` is still required for the RaspberryPi5 image and in fact, it is not. Thus, removed that line from the comment. So please merge accordingly.

Reply:
@jens-maus for a different use case, I've just used `enable_uart=1` on Raspberry Pi 5, but this did not give me any UART :thinking: Did you had to do something additional to get this going on Raspberry Pi 5?